### PR TITLE
fix: add OleanjiKingCode

### LINKED ISSUE #1405: Adebayo Olamilekan
## Profile Screenshot

image

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
resolves #1405

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/OleanjiKingCode.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75235148/193233146-93bb266c-9ad7-4021-970b-96cb303501b1.png",
```
Comment (Panquesito7) [inline profiles/OleanjiKingCode.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/193316450-19e6aa4f-a0a6-44b5-8e2e-fec79dc581e6.png",
```

File: profiles/OleanjiKingCode.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "issueId": 1405,
+  "name": "Adebayo Olamilekan",
+  "username": "OleanjiKingCode"
```
Comment (SurbhiMotwani23) [inline profiles/OleanjiKingCode.json:5]: "issueId": 1405,
  "name": "Adebayo Olamilekan",
  "username": "OleanjiKingCode"

**Review by krshkodes — APPROVED** [review]
I would say that the image doesn't matter much, but would definitely recommend Panquesito's suggestion

**Comment by Pradumnasaraf** [comment]
Closing PR due to inactivity. Feel free to raise a new one.

**Pull request closed without merging**